### PR TITLE
chore(destroy): add current path `.` as the default path

### LINKED ISSUE #420: bug: `envd destroy` without any flags gets error
## Description

```bash
$ envd destroy
error: One of the following flags is required: --path and --name.
```

However, from the doc: https://envd.tensorchord.ai/docs/build-envd/#hello-world, the command of "destroy the environment" no use any of these flags.

## Reproduction

```bash
envd up
envd destroy
```

## Additional Info

```
envd: v0.1.0-alpha.12+8531491
  BuildDate: 2022-06-17T01:20:35Z
  GitCommit: 853149189d88e6ecf6a5a924e3aa19d8f7993f7e
  GitTreeState: clean
  GoVersion: go1.18.3
  Compiler: gc
  Platform: darwin/amd64
```

I recommend adding a default path (`.`) in the `destroy` CLI command.

### PR DESCRIPTION
**Pull request by Triple-Z**
Signed-off-by: Triple-Z <me@triplez.cn>

closed #420

### CODE REVIEW

**Comment by Triple-Z** [comment]
This PR will have the following outputs.

```
$ ~/Code/envd/bin/envd destroy --help
NAME:
   envd destroy - destroys the envd environment

USAGE:
   envd destroy [command options] [arguments...]

OPTIONS:
   --name value, -n value  Name of the environment
   --path value, -p value  Path to the directory containing the build.envd (default: .)
```

BTW, maybe `down` is a better word than `destroy`:
1. `docker-compose` uses it.
2. `down` is the antonym for `up`.
3.  `destroy` is easy to typo :(

File: pkg/app/destroy.go [hunk 1]
```diff
@@ -32,9 +32,11 @@ var CommandDestroy = &cli.Command{
...
 	Flags: []cli.Flag{
 		&cli.PathFlag{
-			Name:    "path",
-			Usage:   "Path to the directory containing the build.envd",
-			Aliases: []string{"p"},
+			Name:        "path",
+			Usage:       "Path to the directory containing the build.envd",
+			Aliases:     []string{"p"},
+			DefaultText: ".",
+			Value:       ".",
```
Comment (terrytangyuan) [inline pkg/app/destroy.go:39]: When --name is specified, there will be an error since both flags have values.

Comment (Triple-Z) [reply pkg/app/destroy.go]: That's a problem, I have fixed it by assigning the default value to `path` when both flags don't have values.

**Review by terrytangyuan — APPROVED** [review]

**Comment by gaocegege** [comment]
Thanks for your contribution! :tada: :+1:

**Comment by gaocegege** [comment]
@all-contributors Please add @Triple-Z for his code contributions!

**Pull request merged**